src/sections/greenNav.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex justify-between p-4 text-primaryGreen">
+    <div class="flex flex-col text-smallerSize md:text-2xl"><h1>DE</h1><p>EN</p></div>
+    <div class=" text-3xl lg:text-5xl font-bold relative">
+        <img src="/src/sections/img/Vector-5.webp" alt="menu-image" class=" md:hidden lg:hidden w-48 h-14">
```

Comment:
This space should be removed.
 I see that you want to hide it on medium and large screens, could you try to apply it only to medium and check what will happen to large screens too?